src/store_tests.cc

```diff
@@ -410,3 +217,162 @@ bool test_init_and_recover_containers(bool print_all) {
   return true;
 }
 
+bool test_policy_store(bool print_all) {
```

Comment:
I think this single large test case handling all sub-ops is difficult to follow and to maintain.

It's ok if you wish to go in like as-is, ... but I think we should talk and I am happy to refactor this into smaller unit-test cases, each for one single operation and its api.

For instance the new code from L352 onwards, `ps.Serialize(&saved))` ... can all go into a sub-case `test_policy_store_serialize_deserialize()` function.